tests/console/yast2_settings.pm

```diff
@@ -0,0 +1,89 @@
+# SUSE's openQA tests
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: YaST writes config to 70-yast.conf and detects conflicts
+# - Install yast2-network
+# - verify writing config to the correct file
+# - ensure conflict detected when custom config with higher priority exists
+# - validate that YaST applies changes to the system. bsc#1167234
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base "y2_module_consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use y2lan_restart_common;
+
+my $yastconf              = '/etc/sysctl.d/70-yast.conf';
+my $customconf_in_etc     = '/etc/sysctl.d/90-custom.conf';
+my $customconf_in_usr_lib = '/usr/lib/sysctl.d/90-custom.conf';
+my $ipv4_forward          = 'net.ipv4.ip_forward';
+
+sub verify_write_config {
+    open_yast2_routing();
+    send_key 'spc';
+    assert_screen "ipv4_forwarding_enabled";
+    send_key 'alt-n';
+    wait_serial("yast2-routing-status-0", 180) || die "'yast2-routing' didn't finish";
+}
+
+sub ensure_conflict_detected {
+    _create_conflicts();
+    open_yast2_routing();
+    send_key 'spc';
+    assert_screen "ipv4_forwarding_disabled";
+    send_key 'alt-n', wait_screen_change => 1;
+    assert_screen "sysctl_ip_forward_conflict";
+    #assert_script_run("grep 'have conflicts with' /var/log/YaST2/y2log";
```

Comment:
could you please delete this dead line code?

Reply:
Done